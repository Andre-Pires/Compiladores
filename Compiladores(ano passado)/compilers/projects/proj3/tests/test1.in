b = b + 1
a = b + c
d = c
e = a + b
