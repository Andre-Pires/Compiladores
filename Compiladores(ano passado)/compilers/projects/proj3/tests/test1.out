 Number of Registers: 4
	r2 = ARP + 8
	r2 = *r2
	r2 = r2 + 1
	r1 = ARP + 12
	r1 = *r1
	r0 = r2 + r1
	r1 = r1
	r3 = ARP + 4
	*r3 = r0
	r0 = r0 + r2
	r3 = ARP + 20
	*r3 = r0
	r3 = ARP + 16
	*r3 = r1
	r3 = ARP + 8
	*r3 = r2
 Number Load Operations: 2
 Number Store Operations: 4
